subprojects/core/src/main/java/org/gradle/api/internal/TaskOutputsEnterpriseInternal.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2010 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.internal;
+
+import org.gradle.api.NonNullApi;
+
+@NonNullApi
+public interface TaskOutputsEnterpriseInternal extends TaskOutputsInternal {
```

Comment:
:x: We should describe why these methods are not on `TaskOutputsInternal` here.

Reply:
Done in 6f1d0728d6f4ea6ab55a751949e6298756aa51b9.